### PR TITLE
Override input autofill color to dark blue

### PR DESCRIPTION
**Pull request by inesilva**

### CODE REVIEW

**Comment by davidhajba** [comment]
hey @inesilva 
when I try the autofill on the 'Deploy Preview ready!' it still changes to yellow for me on Safari

File: css/_stay-up-to-date.scss [hunk 1]
```diff
@@ -85,6 +85,10 @@
         font-family: $main-font-thin;
         box-shadow: 0 25px 35px 0 $shadow-color;
       }
+      
+      input:auto-fill {
+        background-color: $dark-blue;
+      }
```
Comment (oscard0m) [inline css/_stay-up-to-date.scss:91]: I think you need `input:-webkit-autofill` too:
https://developer.mozilla.org/en-US/docs/Web/CSS/:autofill#examples

**Review by oscard0m — CHANGES_REQUESTED** [review]

**Comment by oscard0m** [comment]
This is the native behaviour of the browser, btw.

**Comment by davidhajba** [comment]
`This is the native behaviour of the browser, btw. `

I see, thank you, I guess just leave it as it is I guess - right?

**Comment by oscard0m** [comment]
> `This is the native behaviour of the browser, btw. `
> 
> I see, thank you, I guess just leave it as it is I guess - right?

I think we don't want to focus on that level of detail for now unless is an easy fix.

**Review by oscard0m — COMMENTED** [review]
I still see the yellow in Safari when using Autofill to hide my email. Is it working for you @inesilva

**Comment by oscard0m** [comment]
Apparently a CSS hack was necessary 😱 
https://stackoverflow.com/a/75295943/3237248

**Comment by davidhajba** [comment]
🎉 🎉 🎉
great, thank you

**Review by davidhajba — APPROVED** [review]

**Review by oscard0m — APPROVED** [review]

**Pull request merged**